2025-04-11 11:58:29.03
f4e67d48
User
c = connection.execute(
2025-04-11 11:58:29.03
f4e67d48
User
^^^^^^^^^^^^^^^^^^^
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1416, in execute
2025-04-11 11:58:29.03
f4e67d48
User
return meth(
2025-04-11 11:58:29.03
f4e67d48
User
^^^^^
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/sql/elements.py", line 523, in _execute_on_connection
2025-04-11 11:58:29.03
f4e67d48
User
return connection._execute_clauseelement(
2025-04-11 11:58:29.03
f4e67d48
User
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1638, in _execute_clauseelement
2025-04-11 11:58:29.03
f4e67d48
User
ret = self._execute_context(
2025-04-11 11:58:29.03
f4e67d48
User
^^^^^^^^^^^^^^^^^^^^^^
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1843, in _execute_context
2025-04-11 11:58:29.03
f4e67d48
User
return self._exec_single_context(
2025-04-11 11:58:29.03
f4e67d48
User
^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1983, in _exec_single_context
2025-04-11 11:58:29.03
f4e67d48
User
self._handle_dbapi_exception(
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 2355, in _handle_dbapi_exception
2025-04-11 11:58:29.03
f4e67d48
User
raise exc_info[1].with_traceback(exc_info[2])
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 1933, in _exec_single_context
2025-04-11 11:58:29.03
f4e67d48
User
self.dialect.do_executemany(
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 800, in do_executemany
2025-04-11 11:58:29.03
f4e67d48
User
cursor.executemany(statement, parameters)
2025-04-11 11:58:29.03
f4e67d48
User
File "/nix/store/clx0mcir7qw8zk36zbr4jra789g3knf6-python3-3.11.10/lib/python3.11/encodings/utf_8.py", line 15, in decode
2025-04-11 11:58:29.03
f4e67d48
User
def decode(input, errors='strict'):
2025-04-11 11:58:29.03
f4e67d48
User
2025-04-11 11:58:29.03
f4e67d48
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/gunicorn/workers/base.py", line 204, in handle_abort
2025-04-11 11:58:29.03
f4e67d48
User
sys.exit(1)
2025-04-11 11:58:29.03
f4e67d48
User
SystemExit: 1